src/Configuration/Cache.php

```diff
@@ -101,6 +102,35 @@ class Cache extends ConfigurationAnnotation
      */
     private $staleIfError;
 
+    public function __construct(
+        $values = [],
+        $expires = null,
+        $maxage = null,
+        $smaxage = null,
+        $public = false,
+        $mustRevalidate = false,
+        $vary = [],
+        $lastModified = null,
+        $Etag = null,
+        $maxstale = null,
+        $staleWhileRevalidate = null,
+        $staleIfError = null
+    ) {
```

Comment:
Please add parameter types where possible on PHP 7.2. Please add a `@param` annotation if you cannot define a property type.

Same with other constructors in this PR.